tests/parsers/agilent_gen5/to_allotrope_test.py

```diff
@@ -1,35 +1,64 @@
+from typing import Any
+
 import pytest
 
+from allotropy.allotrope.allotrope import serialize_allotrope
+from allotropy.allotrope.models.ultraviolet_absorbance_benchling_2023_09_ultraviolet_absorbance import (
+    Model,
+)
 from allotropy.exceptions import AllotropeConversionError
 from allotropy.parser_factory import Vendor
-from tests.parsers.test_utils import from_file, validate_contents, validate_schema
+from tests.parsers.test_utils import (
+    from_file,
+    model_from_file,
+    validate_contents,
+    validate_schema,
+)
 
 VENDOR_TYPE = Vendor.AGILENT_GEN5
 
+ABSORBENCE_FILENAMES = [
+    "endpoint_pathlength_correct_singleplate",
+    "endpoint_stdcurve_singleplate",
+    "endpoint_stdcurve_singleplate_2",
+    "endpoint_stdcurve_multiplate",
+    "kinetic_helper_gene_growth_curve",
+    "kinetic_singleplate",
+    "kinetic_multiplate",
+]
 
-@pytest.mark.parametrize(
-    "filename",
-    [
-        "endpoint_pathlength_correct_singleplate",
-        "endpoint_stdcurve_singleplate",
-        "endpoint_stdcurve_singleplate_2",
-        "endpoint_stdcurve_multiplate",
-        "kinetic_helper_gene_growth_curve",
-        "kinetic_singleplate",
-        "kinetic_multiplate",
-    ],
-)
+
+def _validate_allotrope_dict(
+    allotrope_dict: dict[str, Any], expected_filepath: str
+) -> None:
+    validate_schema(
+        allotrope_dict,
+        "ultraviolet-absorbance/BENCHLING/2023/09/ultraviolet-absorbance.json",
+    )
+    validate_contents(allotrope_dict, expected_filepath)
+
+
+@pytest.mark.parametrize("filename", ABSORBENCE_FILENAMES)
 def test_to_allotrope_absorbance(filename: str) -> None:
     test_filepath = f"tests/parsers/agilent_gen5/testdata/absorbance/{filename}.txt"
     expected_filepath = (
         f"tests/parsers/agilent_gen5/testdata/absorbance/{filename}.json"
     )
     allotrope_dict = from_file(test_filepath, VENDOR_TYPE)
-    validate_schema(
-        allotrope_dict,
-        "ultraviolet-absorbance/BENCHLING/2023/09/ultraviolet-absorbance.json",
+    _validate_allotrope_dict(allotrope_dict, expected_filepath)
+
+
+# Test allotrope_model_from_file().
+@pytest.mark.parametrize("filename", ABSORBENCE_FILENAMES)
```

Comment:
`allotrope_model_from_file` is tested implicitly by calls to `allotrope_from_io`. 

I'm fine with adding a test for it if you really want, but I don't think we need to run on all test cases here.

Reply:
Updated PR description. The function currently has no calls, so I figured it should be tested. Fine just running on one test case.